### DISCUSSION TITLE
chaos-mesh NetworkChaos delay issues

### DISCUSSION THREAD

**Discussion by oldthreefeng** [chaos-mesh/chaos-mesh#4279] [Q&A]
for kubernetes, we have livenessProbe & readinessProbe .  default set it to timeoutSeconds 1s. 

when we inject chaos-mesh network latency > 1000ms in the pods .  our pod will be killed by kubelet health , can not act as our expect .

can chaos-mesh team solve this ? 
ignore some path for latency ?

**Comment by wang-xiaowu** [comment]
same question here.

**Reply by oldthreefeng** [reply]
our health check has its own logic while network delay .
